src/main/java/org/jabref/gui/dialogs/AutosaveUIManager.java

```diff
@@ -25,10 +29,20 @@ public AutosaveUIManager(BasePanel panel) {
 
     @Subscribe
     public void listen(@SuppressWarnings("unused") AutosaveEvent event) {
-        try {
-            new SaveDatabaseAction(panel, Globals.prefs, Globals.entryTypesManager).save(SaveDatabaseAction.SaveDatabaseMode.SILENT);
-        } catch (Throwable e) {
-            LOGGER.error("Problem occurred while saving.", e);
+        if (timer != null) {
+            timer.cancel();
         }
+
+        timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                try {
+                    new SaveDatabaseAction(panel, Globals.prefs, Globals.entryTypesManager).save(SaveDatabaseAction.SaveDatabaseMode.SILENT);
+                } catch (Throwable e) {
+                    LOGGER.error("Problem occured while saving.", e);
+                }
+            }
+        }, 200);
```

Comment:
Please introduce a constant - I think, at the header of this class is OK - (no magic numbers in the code)